### PR TITLE
:sparkles: added support for superwise.ai in docs

### PR DESCRIPTION
**Pull request by barakbl**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -423,6 +423,253 @@ You can monitor the progress via the SageMaker UI console. Here is an example of
 
 ## Monitor ML Models in Production
 
+## Superwise.ai
+
+Superwise provide organization to streamline the model observability and monitoring process 🚀
+It's platform agnostic and provide an extensive freemium tier.
+
+In this part, you'll integrate with [Superwise.ai](https://www.superwise.ai/) gain full visibility on your deployed models.
```
Comment (pm3310) [inline docs/index.md:431]: Missing `to` -> `integrate with [Superwise.ai](https://www.superwise.ai/) to gain full visibility`

File: docs/index.md [hunk 2]
```diff
@@ -423,6 +423,253 @@ You can monitor the progress via the SageMaker UI console. Here is an example of
...
+
+Superwise provide organization to streamline the model observability and monitoring process 🚀
+It's platform agnostic and provide an extensive freemium tier.
+
+In this part, you'll integrate with [Superwise.ai](https://www.superwise.ai/) gain full visibility on your deployed models. 
+once integrated you can use Superwise platform to monitor and define workflows to detect and handle: data drift, performance degregation, data integrity, model activity or any other customized monitoring use case. 
+
+### Step 1: Create Superwise Account
+
+Go to [Superwise](https://portal.superwise.ai) and create an account using the Account button. The free tier support up to 3 models.
```
Comment (pm3310) [inline docs/index.md:436]: `support up` -> `supports up`

File: docs/index.md [hunk 3]
```diff
@@ -423,6 +423,253 @@ You can monitor the progress via the SageMaker UI console. Here is an example of
...
+    features_test = features_test_df.values
+    labels_test = labels_test_df.values.ravel()
+
+
+    clf = SVC(gamma='auto', kernel="linear")
+    clf.fit(features_train, labels_train)
+
+    ###### Report Testing Data ######
+    test_predictions = clf.predict(features_test)
+
```
Comment (pm3310) [inline docs/index.md:533]: You can remove a couple of blank lines

File: docs/index.md [hunk 4]
```diff
@@ -423,6 +423,253 @@ You can monitor the progress via the SageMaker UI console. Here is an example of
...
+    accuracy = accuracy_score(labels_test, test_predictions)
+
+    output_model_file_path = os.path.join(model_save_path, 'model.pkl')
+    joblib.dump(clf, output_model_file_path)
+
+    accuracy_report_file_path = os.path.join(model_save_path, 'report.txt')
+    with open(accuracy_report_file_path, 'w') as _out:
+        _out.write(str(accuracy))
+
+
```
Comment (pm3310) [inline docs/index.md:545]: You can remove a couple of blank lines

File: docs/index.md (hunk 1 above)
Comment (barakbl) [reply docs/index.md]: fixed

**Review by pm3310 — APPROVED** [review]

**Pull request merged**